api/routes.py

```diff
@@ -672,6 +672,18 @@ def track_analytics_event(identifier_type, identifier, event_type):
     )
 
 
+@library_route(
+    "/playtimes/<collection_name>/<identifier_type>/<path:identifier>", methods=["POST"]
```

Comment:
I might be better to use `collection_id`, rather than `collection_name`, for the first component of the path, since it is less likely to change.